packer/jenkins-agent-MacOS-x64.json

```diff
@@ -0,0 +1,81 @@
+{
+  "variables":{
+    "ami_name":"Jenkins-Agent-MacOS-X64-Mac1Metal",
+    "os_version": "12.4",
+    "os_architecture": "x86_64_mac",
+    "build-region":"us-east-1",
+    "build-vpc":"vpc-<>",
+    "build-subnet":"subnet-<>",
+    "build-secgrp":"sg-<>",
```

Comment:
Are we going to replace this or is this how it is supposed to be?

Reply:
These values will be replaced per specific account.